File: seed/go-sdk/examples/always-send-required-properties/dynamic-snippets/example17/snippet.go
package example

import (
    client "github.com/examples/fern/client"
    option "github.com/examples/fern/option"
    context "context"
    fern "github.com/examples/fern"
)

func do() () {
    client := client.NewClient(
        option.WithToken(
            "<token>",
        ),
    )
    client.Service.GetMetadata(
        context.TODO(),
        &fern.GetMetadataRequest{
            Shallow: fern.Bool(
                true,
            ),
            Tag: []*string{
                fern.String(
                    "tag",
                ),
            },
            XApiVersion: "X-API-Version",
        },
    )}

File: seed/ts-sdk/examples/examples-with-api-reference/src/dynamic-snippets/example10/snippet.ts
import { SeedExamplesClient } from "../..";

async function main(): Promise<void> {
    const client = new SeedExamplesClient({
        environment: "https://api.fern.com",
        token: "<token>",
    });
    
    await client.health.service.ping();
}
main();
